docs/_documentations/installoncloud.md

```diff
@@ -101,7 +101,7 @@ For more information about Tekton, see [Getting started with the Tekton Dashboar
 ## After installing Codewind
 
 ### Setting the Docker registry
-Upon creating a Codewind workspace. The container registry to deploy your projects to must be set. When you go to create or add an existing project to Codewind, Codewind will prompt you for the registry. See (Docker registry docs) for guidance on using proper container registries
+After creating a Codewind workspace. The container registry to deploy your projects to must be set. When you go to create or add an existing project to Codewind, Codewind will prompt you for the registry. See [Docker registry docs](https://www.eclipse.org/codewind/dockerregistry.html) for guidance on using proper container registries
```

Comment:
Sorry I missed this--could you add a period at the end of the last sentence of line 104?